### DISCUSSION TITLE
How can I create custom breakpoint instead of using ftd.device

### DISCUSSION THREAD

**Discussion by Sarvom** [fastn-stack/fastn#1468] [Q&A]
I am building a dashboard where I need to use custom breakpoint to make the page responsive, How can I create one?

Can I use this?

https://github.com/fastn-stack/fastn/blob/main/Cheatsheet.md#ftdbreakpoint-width

**Comment by Sarvom** [comment]
If I could define a custom breakpoint like this,

`-- ftd.breakpoint-width-data $my-breakpoint: 1284`

And use it in similar

```
-- ftd.column:
overflow-x if { ftd.breakpoint-check(my-breakpoint) }: auto 
```

**Comment by Heulitig** [comment]
@Sarvom We are working on this. We will update you regarding this soon.

**Reply by amitu** [reply]
@Heulitig please share the how-to here as a new answer and ask @Sarvom to accept it as an answer if it works for him.

**Comment by amitu** [comment]
@Heulitig can you write a blog post about this as well?
